maddymz/irondb
# Us379

US resolves all ESLint errors, updates the auth flow with a place holder for user/admin-panel, and a basic first interpretation of Base page of the data-entry route.

Steps to test once on repo branch post pull:

1. *turn on docker
2. Run `./iron.sh`
3. Select `3`
4.  Navigate browser to `localhost:3001/register`
![screen shot 2019-01-15 at 2 52 23 pm](https://user-images.githubusercontent.com/6512755/51207202-2c7e7b80-18d8-11e9-989d-76a38cca9754.jpg)

5. Register a user, browser should be redirected to `localhost:3001/panel`
![screen shot 2019-01-15 at 2 52 41 pm](https://user-images.githubusercontent.com/6512755/51207231-3d2ef180-18d8-11e9-8226-5e099d6a31f3.jpg)

6. Select navbar link for data-entry
![screen shot 2019-01-15 at 2 52 50 pm](https://user-images.githubusercontent.com/6512755/51207249-46b85980-18d8-11e9-89c5-2bcdb9dffdd7.jpg)

7. Check that both links give the corresponding urls `/data-entry/tool` and `/data-entry/editor` with 404 error pages.
8. Logout
9. NAvigate to `localhost:3001/data-entry`
Should get a 401 error.
10. on terminal run `npm run pretest` for ESLint report, result should be:
![screen shot 2019-01-15 at 3 19 28 pm](https://user-images.githubusercontent.com/6512755/51207506-fa214e00-18d8-11e9-8fad-15947ea7f6d6.jpg)
